### DISCUSSION TITLE
pulsar-io & geo-replication can not work without system-topic synchroniztion

### DISCUSSION THREAD

**Discussion by wuyanzan** [apache/pulsar#20849] [Q&A]
As far as i know, geo-replication will not deal with system-topic.

so here comes the problem: 
how should we process the failure of cluster between different Area with function?

for example:
Beijing & Shanghai,  we have two cluster using geo-replication to do data synchronization，unfortunately, Beijing out of service some day.
how should we deliver the function/sink/source to Shanghai  cause the information store in the function-topics.

**Comment by poorbarcode** [comment]
@wuyanzan 

The topic maintaining function metadata is `public/functions/metadata` by default. I don't see any restrictions to prevent enabling Geo-Replication for this topic. Could you please share a link with me?
